cr_module/classes/redfish.py

```diff
@@ -226,7 +227,51 @@ def restore_session_from_file(self):
         self.session_was_restored = True
 
         return
-
+    
+    def remove_session_lock(self):
+        if (self.cli_args.sessionlock is False or self.session_file_path is None):
+            return
+        try:
+            if os.path.exists(self.session_file_lock):
+                os.remove(self.session_file_lock)
+        except Exception as e:
+            self.exit_on_error(f"Unable to remove session lock '{self.session_file_lock}': {e}", "CRITICAL")
+        return
+    
+    def is_session_locked(self):
+        if (self.cli_args.sessionlock is False or self.session_file_path is None):
+            return False
+        if os.path.exists(self.session_file_lock) is False:
+            return False
+        return os.path.exists(self.session_file_lock)
```

Comment:
Any code after this lines will never be executed.

Reply:
Glad you saw that. I did some testing and the PID only works using a user with sudo rights. So I just fully changed it to timestamps. This should only be a fallback if for some reason the lock is not getting cleared. This should not happen.